clang/lib/AST/ByteCode/InterpBuiltin.cpp

```diff
@@ -3412,6 +3412,13 @@ bool InterpretBuiltin(InterpState &S, CodePtr OpPC, const CallExpr *Call,
   case Builtin::BI__builtin_elementwise_fma:
     return interp__builtin_elementwise_fma(S, OpPC, Call);
 
+  case clang::X86::BI__builtin_ia32_pblendvb128:
+  case clang::X86::BI__builtin_ia32_pblendvb256:
+    return interp__builtin_elementwise_triop(
+        S, OpPC, Call, [](const APSInt &F, const APSInt &T, const APSInt &C) {
+          return ((APInt)C).isNegative() ? T : F;
```

Comment:
Static analysis flagged that this lambda will return by copy, I spent some time looking at the code and it was not clear to me that was intended or not. The fix would be to use tailing return type and return a const ref.

Reply:
So was returning by copy intended? Can you please confirm or if this is a bug please put in a fix? @RKSimon